sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/synthetic/README.md

```diff
@@ -0,0 +1,302 @@
+# Usage Guide
```

Comment:
This content should be moved into the root readme. Customers are unlikely to find it here, and it wont be published anywhere alongside the ref docs/pypi etc.

Reply:
it has been and removed